packages/@okta/vuepress-site/docs/guides/protect-your-api/main/springboot/request.md

```diff
@@ -0,0 +1,7 @@
+```http
+https://localhost:8080/api/whoami Bearer: ${TOKEN}
+```
+
+```http
+https://localhost:8080/api/hello
+```
```

Comment:
```suggestion
```bash
curl --header "Authorization: Bearer: ${TOKEN}" https://localhost:8080/api/whoami
```

```bash
curl https://localhost:8080/api/hello
```
```

Reply:
The previous version of this was missing the header name `Authorization` and wasn't clear in the rendered version

Reply:
I'm not convinced we need this cURL example. The above text does tell the reader to use postman for testing, and if we added a cURL example here we'd have to add it to the other guides as well. I am happy to consider this as a future improvement.

Reply:
Uggh, Postman, eh? HTTPie is so much better than both Postman and cURL!

Reply:
We do have the curl example on other guides as well